Yoast/wordpress-seo
# Add cornerstone to state

## Summary

This PR can be summarized in the following changelog entry:

* Add cornerstoneContent boolean to redux store.

## Test instructions

This PR can be tested by following these steps:

* Checkout branch and run `yarn install` and `yarn test` (when the tests have passed CR you can assume they are fine, though do check them).
* You could expose the store and action to the window in `edit.js`, and dispatch (`store.dispatch( action() )` and see whether the state was changed correctly.

## Quality assurance


Fixes #9328 

_Note I accidentally misnamed the branch (issuenumber). Figured that was too minor to go to the trouble of fixing._
